Dockerfile

```diff
@@ -28,7 +28,8 @@ WORKDIR "/opt"
 # ESB Setup
 COPY Packages/wso2esb-${ESB_VERSION}.zip /opt
 RUN unzip /opt/wso2esb-${ESB_VERSION}.zip 
-RUN chmod +x /opt/wso2esb-${ESB_VERSION}/bin/wso2server.sh
+RUN chmod +x /opt/wso2esb-${ESB_VERSION}/bin/wso2server.sh && \
```

Comment:
you want to delete the zip in the previous layer (after the unzip), otherwise the archive still takes space in the previous layer and you don't get any benefit.